numpy/core/src/umath/ufunc_object.c

```diff
@@ -3063,6 +3063,8 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
     Py_XDECREF(axis);
     Py_XDECREF(full_args.in);
     Py_XDECREF(full_args.out);
+    PyArray_free(remap_axis_memory);
```

Comment:
Darn, that's quite dumb of me, having it in the failure path but not in the main one.